### DISCUSSION TITLE
check the training loss when using gluonts

### DISCUSSION THREAD

**Discussion by kkckk1110** [awslabs/gluonts#3071] [Q&A]
Hello, I am trying to train a DeepAR model with gluonts. I hope to check the loss of each epoch during the training process. As suggested by the docs, I tried the mxnet-based approach. Unfortunately, the mxnet cannot be applicable in my device (I am using a M1 Macbook). Therefore, I wonder that, is there any other method that I can use to examine the training loss? Thanks a lot!
